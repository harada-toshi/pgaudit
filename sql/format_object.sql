\! cp sql/format_object/format_object.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl -w restart -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! sh sql/format_object/format_object.sh
