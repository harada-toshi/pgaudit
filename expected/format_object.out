	        format = 'AUDIT: %class,%object_type,%object_id,%command_tag,%command_text'
LOG:  AUDIT: DDL,TABLE,public.foo,CREATE TABLE,CREATE TABLE foo (id int, data text);
LOG:  AUDIT: DDL,INDEX,public.foo_id_idx,CREATE INDEX,CREATE INDEX foo_id_idx ON foo USING btree (id);
LOG:  AUDIT: DDL,VIEW,public.foo_v,CREATE VIEW,CREATE VIEW foo_v AS SELECT * FROM foo;
LOG:  AUDIT: READ WRITE,TABLE MATERIALIZED VIEW,public.foo public.foo_mv,SELECT INSERT,CREATE MATERIALIZED VIEW foo_mv AS SELECT * FROM foo;
LOG:  AUDIT: DDL,MATERIALIZED VIEW,public.foo_mv,CREATE MATERIALIZED VIEW,CREATE MATERIALIZED VIEW foo_mv AS SELECT * FROM foo;
LOG:  AUDIT: DDL,SEQUENCE,public.dummy_seq,CREATE SEQUENCE,CREATE SEQUENCE dummy_seq;
LOG:  AUDIT: WRITE,TABLE,public.foo,INSERT,INSERT INTO foo VALUES (1, 'aaa'),(2,'bbb');
LOG:  AUDIT: MISC MISC READ,MATERIALIZED VIEW MATERIALIZED VIEW TABLE,public.foo_mv public.foo_mv public.foo,UNKNOWN UNKNOWN SELECT,REFRESH MATERIALIZED VIEW foo_mv;
LOG:  AUDIT: DDL,MATERIALIZED VIEW,public.foo_mv,REFRESH MATERIALIZED VIEW,REFRESH MATERIALIZED VIEW foo_mv;
LOG:  AUDIT: MISC,,,REINDEX,REINDEX INDEX foo_id_idx;
LOG:  AUDIT: MISC,,,REINDEX,REINDEX TABLE foo;
LOG:  AUDIT: READ,TABLE,public.foo,SELECT,TABLE foo;
LOG:  AUDIT: READ MISC READ,VIEW VIEW TABLE,public.foo_v public.foo_v public.foo,SELECT UNKNOWN SELECT,TABLE foo_v;
LOG:  AUDIT: READ,MATERIALIZED VIEW,public.foo_mv,SELECT,TABLE foo_mv;
LOG:  AUDIT: DDL,SEQUENCE,public.dummy_seq,DROP SEQUENCE,DROP SEQUENCE dummy_seq;
LOG:  AUDIT: DDL,MATERIALIZED VIEW,public.foo_mv,DROP MATERIALIZED VIEW,DROP MATERIALIZED VIEW foo_mv;
LOG:  AUDIT: DDL,VIEW,public.foo_v,DROP VIEW,DROP VIEW foo_v;
LOG:  AUDIT: DDL,INDEX,public.foo_id_idx,DROP INDEX,DROP INDEX foo_id_idx;
LOG:  AUDIT: DDL,TABLE,public.foo,DROP TABLE,DROP TABLE foo;
